MapboxSearch/base/src/main/java/com/mapbox/search/base/location/LocationEngineAdapter.kt

```diff
@@ -54,11 +68,21 @@ class LocationEngineAdapter(
 
     private fun startLocationListener() {
         locationProvider?.addLocationObserver(locationObserver)
+
+        locationEngineObservationTimeout?.let { timeout ->
```

Comment:
Would be great to cover it with tests

Reply:
It's kind of a crutch that's needed because of the underlying design. I don't think we should solidify it.